.github/workflows/pika.yml

```
@@ -84,59 +84,49 @@
-      - name: Install deps
+      - name: use SCL
         run: |
-          dnf update -y
-          dnf install -y bash cmake wget git autoconf gcc perl-Digest-SHA tcl which tar g++ tar epel-release gcc-c++ libstdc++-devel gcc-toolset-13
-
-      - name: Set up Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: 1.19
+          yum install centos-release-scl
+          yum install devtoolset-9  
```

yamllint

[error] trailing spaces

(trailing-spaces)